static/js/biomedical/protein/protein_protein_interaction_graph.tsx

```diff
@@ -88,13 +98,34 @@ export class ProteinProteinInteractionGraph extends React.Component<
   }
 
   render(): JSX.Element {
-    return <div id={CHART_ID}></div>;
+    if (this.state.graphData === null) {
+      return <div></div>;
+    }
+    return (
+      <>
+        <div id={CHART_ID}></div>
+        <FormGroup>
+          <Label for={DEPTH_INPUT_ID}>Depth</Label>
+          <Input
+            id={DEPTH_INPUT_ID}
+            className={DEPTH_INPUT_ID}
```

Comment:
I don't think it's necessary to set both the id and className here, you could just use the id to select for the element

Reply:
Thanks for the review! Got it will fix!